### PR TITLE
Add basic flask structure for the app

### PR DESCRIPTION
**Pull request by carpecodeum**
# Description

Added basic flask structure to port the app to flask.
In order to reproduce the changes:
1. make a venv directory inside labellab-flask
2. cd venv
3. start the virtualenv
4. pip install all the dependencies 
5. add details to .envsample
6. flask run

Related to #283  

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?
Screenshot 2020-02-28 at 8 01 46 PM
Screenshot 2020-02-28 at 8 30 02 PM
Screenshot 2020-02-28 at 8 30 14 PM

**Test Configuration**:

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

Also, include screenshots of app for the verification and reviewing purpose.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by carpecodeum** [comment]
@ivantha please review this and tell me your views on it!

File: labellab-flask/app.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import os
```
Comment (UdeshUK) [inline labellab-flask/app.py:1]: Remove unused imports

File: labellab-flask/requirements.txt. [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Click==7.0
+Flask==1.1.1
+Flask-Cors==3.0.8
+itsdangerous==1.1.0
+Jinja2==2.11.1
+MarkupSafe==1.1.1
+python-dotenv==0.11.0
+six==1.14.0
+Werkzeug==1.0.0
```
Comment (ivantha) [inline labellab-flask/requirements.txt.:9]: Shall we add the packages as necessary?

Comment (carpecodeum) [reply labellab-flask/requirements.txt.:9]: These were added when I had set-uped the project. I think we should keep them as they can come in handy as the development goes forward.

File: labellab-flask/app.py (hunk 1 above)
Comment (carpecodeum) [reply labellab-flask/app.py:1]: sure @UdeshUK

**Pull request merged**